lib/factory_girl/linter.rb

```diff
@@ -10,17 +10,15 @@ def initialize(factories_to_lint)
     end
 
     def lint!
-      if invalid_factories.any?
+      if @invalid_factories.any?
         raise InvalidFactoryError, error_message
       end
     end
 
     private
 
-    attr_reader :factories_to_lint, :invalid_factories
```

Comment:
Rather than removing the private `attr_readers` and falling back to instance variables, the ([silly](https://bugs.ruby-lang.org/issues/10967)) warning can be worked around like this:

```ruby
# …public methods…

attr_reader :factories_to_lint, :invalid_factories
private     :factories_to_lint, :invalid_factories

private

# …private methods…
```